topics/first-tech-challenge/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: ftc, first-ftc, ftc-robotics, ftc-sdk
+created_by: FIRST
+display_name: FIRST Tech Challenge
+github_url: https://github.com/ftctechnh
+logo: ftc-logo.png
```

Comment:
Need to fix up this name: https://travis-ci.org/github/explore/builds/308641962#L535-L544